broadinstitute/ddp-angular
# DDP-4233: Angular email question - second iteration

Suggestions and improvements implemented - https://github.com/broadinstitute/ddp-angular/pull/13#issuecomment-568429387

If you want to test it out:
1. run sandbox-app
2. open `http://localhost:4200/text-question`
3. use these parameters:
```
{
    "readonly": false,
    "shown": true,
    "question": "Your child's email address",
    "placeholder": "name@email.com",
    "inputType": "EMAIL",
    "confirmEntry": true,
    "confirmPrompt": "Confirm your child's email address",
    "mismatchMessage": "Please make sure your child's email address matches"
}
```
